src/api/entities/MultiSigProposal/index.ts

```diff
@@ -138,6 +140,13 @@ export class MultiSigProposal extends Entity<UniqueIdentifiers, HumanReadable> {
     const expiry = optionize(momentToDate)(rawExpiry.unwrapOr(null));
     const status = meshProposalStatusToProposalStatus(rawStatus, expiry);
     const autoClose = boolToBoolean(rawAutoClose);
+    const voted: string[] = [];
+    if (votes.length > 0) {
+      votes.forEach(([voteStorageKey, didVote]) => {
+        if (didVote.isTrue && voteStorageKey.args[1].isAccount)
+          voted.push(voteStorageKey.args[1].asAccount.toString());
+      });
+    }
```

Comment:
@polymath-eric I think it's planned to remove signing with identities from Multisigs in the near future https://polymesh.atlassian.net/browse/MESH-2117. For the portal we are only supporting keys as multisig signers so that was part of my logic. I also just didn't consider the existing `signatoryToSignerValue` conversion util. I can make the requested changes if you prefer.

Reply:
Makes sense. Usually the SDK would use`accountIdToString` vs calling `toString`. 

All that does calls `.toString` directly anyway, so not really needed though